content/spin/v2/go-components.md

```diff
@@ -157,12 +184,9 @@ now receive requests in route `/hello`:
 <!-- @selectiveCpy -->
 
 ```bash
-$ curl -i localhost:3000/hello
+$ curl -i localhost:3000
```

Comment:
This no longer matches the manifest (I don't know whether you want to change the route in the manifest or restore it here, but if the former then line 182 will also need a change).

Reply:
Great catch, thank you.
I have updated the `curl` command to make the request to the correct endpoint.